.install-hooks.sh

```
@@ -0,0 +1,16 @@
+#!/bin/sh
+REPO_DIR=$(git rev-parse --show-toplevel)
+GIT_DIR=$REPO_DIR/.git
+VENV_ACTIVATE=$VIRTUAL_ENV/bin/activate
+if [[ ! -f $VENV_ACTIVATE ]]
+then
+    echo "Could not find your virtual environment"
+fi
+
+echo "#!/bin/sh" >> $GIT_DIR/hooks/pre-commit
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)